libcontainer/userns/userns_deprecated.go

```diff
@@ -1,8 +1,13 @@
+// Deprecated: use github.com/moby/sys/userns
 package userns
 
+import "github.com/moby/sys/userns"
+
 // RunningInUserNS detects whether we are currently running in a Linux
 // user namespace and memoizes the result. It returns false on non-Linux
 // platforms.
+//
+// Deprecated: use [userns.RunningInUserNS].
```

Comment:
@thaJeztah I think this should be
```suggestion
// Deprecated: use [github.com/moby/sys/userns.RunningInUserNS].
```

Reply:
Nope; I was wrong.